src/java.desktop/macosx/native/libawt_lwawt/awt/JavaComponentAccessibility.m

```diff
@@ -45,6 +44,12 @@
 #import "JNIUtilities.h"
 #import "AWTView.h"
 
+// these constants are duplicated in CAccessibility.java
+#define JAVA_AX_ALL_CHILDREN (-1)
+#define JAVA_AX_SELECTED_CHILDREN (-2)
+#define JAVA_AX_VISIBLE_CHILDREN (-3)
```

Comment:
Why not mark them as @native in the CAccessibility  and use constants generated by the javac?

Reply:
Done

Reply:
But the code above still exits or it is a github glitch?

Reply:
By convention with @ azuev-java, we try to either not change at all, or change the minimum legacy code.